apis/management.cattle.io/v3/cluster_scan_types.go

```diff
@@ -104,3 +130,22 @@ type CisBenchmarkVersion struct {
 
 	Info CisBenchmarkVersionInfo `json:"info" yaml:"info"`
 }
+
+type ScheduledClusterScanConfig struct {
+	// Cron Expression for Schedule
+	CronSchedule string `yaml:"cronSchedule" json:"schedule,omitempty"`
+	// Number of past scans to keep
+	Retention int `yaml:"retention" json:"retention,omitempty"`
+}
+
+type ScheduledClusterScan struct {
```

Comment:
why is this not ClusterScan with `scheduled` bool? trying to understand why it needs to be called ScheduledClusterScan

Reply:
ScheduledClusterScan is the struct used to store the info at the cluster level. ClusterScan object is the one created when an actual scan is run.